src/main/java/redis/embedded/AbstractRedisInstance.java

```diff
@@ -47,42 +60,99 @@ public InputStream errors() {
     }
 
     private void awaitRedisServerReady() throws IOException, InterruptedException {
+        if (state.isActivating()) {
```

Comment:
I believe here it should be `if (!state.isActivating()) {` ?
I think it's why the tests are failing.